books.js

```diff
@@ -0,0 +1,59 @@
+const titleInput = document.getElementById('title-input');
+const authorInput = document.getElementById('author-input');
+const submitButton = document.getElementById('submit-button');
+let books = [];
```

Comment:
- [ ] I prefer to use `const` here instead of `let`

Reply:
Hello Emmyn, thank you for your feedback. I just wanted to say that due to functionality we cannot use const; and because of the linters, we cannot use var either (Unexpected var, use let or const instead - this is the linter error message),  so we have to use let.